### PR TITLE
Added Font awesome. First story. SASS edit

### PR DESCRIPTION
**Pull request by skirianov**
Please find adjusted packages with react-font-awesome. 

Managed to create first story. The progress will get faster from now on.

### CODE REVIEW

File: src/components/trash/Trash.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+import { Meta } from '@storybook/react';
+
+import Trash from './Trash';
+
+export default {
+  title: 'Trash-icon',
+  component: Trash,
+} as Meta;
+
+export const TrashCan: React.FC = () => (
```
Comment (IgorBolotnikov) [inline src/components/trash/Trash.stories.tsx:11]: It's great that you already know that there are multiple ways to define a React component. `React.FC` does several implicit things which make the typing process more difficult. And also we are using the following notation in other places, so I suggest to keep our style guide uniform across the code base.
```JSX
export const TrashCan = (): React.ReactElement => (
  <Trash bootstrap="btn btn-light" /> // This one should also be changed as per the earlier comment
)
```

File: src/components/trash/Trash.tsx [hunk 2]
```diff
@@ -1,17 +1,22 @@
...
-    return (
-      <div className="trash">
-        <button type="button" className="btn btn-light">
-          <i className="fas fa-trash" />
-        </button>
-      </div>
-    );
-  }
+interface TrashProps {
+  bootstrap: string,
```
Comment (IgorBolotnikov) [inline src/components/trash/Trash.tsx:8]: This attribute better be `className`

File: src/components/trash/Trash.tsx [hunk 3]
```diff
@@ -1,17 +1,22 @@
...
 }
 
+const Trash = ({
+  bootstrap = 'btn btn-light',
+
+}: TrashProps): React.ReactElement => (
+  <div className="trash">
+    <button type="button" className={`${bootstrap}`}>
```
Comment (IgorBolotnikov) [inline src/components/trash/Trash.tsx:16]: Here we can simply pass `className={className}`. Curly braces are used then we need to pass an expression or a variable. Passing strings inside is a bit of an overkill.

File: src/components/trash/trash.sass [hunk 4]
```diff
@@ -3,10 +3,7 @@
 	width: 40px
 	height: 40px
 	background-color: rgb(247, 247, 247)
+	margin: 20px;
```
Comment (IgorBolotnikov) [inline src/components/trash/trash.sass:6]: We don't need punctuation in .sass files so this line should be without semicolon.

**Review by IgorBolotnikov — COMMENTED** [review]
Nice, you've got the grasp of functional components and started with typescript. We need to work on the code style/cleanliness, but that's why we make these reviews. Please reassign me for review once you've resolved all comments.

File: src/components/options/Options.tsx [hunk 5]
```diff
@@ -1,20 +1,21 @@
...
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faTrash } from '@fortawesome/free-solid-svg-icons';
+
+const Options = ():React.ReactElement => {
+  const bootstrap: string = 'btn btn-light';
+
+  return (
+    <div className="row">
+      <div className="options">
+        <Trash bootstrap={`${bootstrap}`} />
```
[the remark this answers is not in the document]
Comment (skirianov) [reply src/components/options/Options.tsx:15]: I was going to re-write Trash.tsx to Icons.tsx, so we can use same component with different props instead Trash.tsx and Edit.tsx respectively. All the styling is same, except for props.

File: src/components/trash/Trash.stories.tsx (hunk 1 above)
Comment (skirianov) [reply src/components/trash/Trash.stories.tsx:11]: OK. Was trying to make the storybook work at least somehow ;D Copy on that.

File: src/components/options/Options.tsx (hunk 5 above)
[the remark this answers is not in the document]
Comment (skirianov) [reply src/components/options/Options.tsx:15]: Copy on that, will do.

File: src/components/trash/Trash.tsx (hunk 3 above)
Comment (skirianov) [reply src/components/trash/Trash.tsx]: Got it, thanks.

**Pull request merged**